const runtimeError = () => {
    // console.log(x)
    let num = 30
    num()
}

module.exports = {
    runtimeError,
};